### ISSUE TITLE
Bundler 2.2.9 breaks installation compared to bundler 2.1.4

### ISSUE THREAD

**Issue by thebravoman** [rubygems/rubygems#4387]
### Describe the problem as clearly as you can

Bundler update from
```
 BUNDLED WITH
-   2.1.4
+  2.2.9
```

### Post steps to reproduce the problem

Call bundle install on my project . No other changes

### Which command did you run?

bundle install

### What were you expecting to happen?

Bundles to be installed

### What actually happened?

```
The gem 'rack' was found in multiple relevant sources.
  * rubygems repository http://localrepo/ or installed locally
  * rubygems repository http://localrepo/, http://rubygems.org/ or installed locally
You must add this gem to the source block for the source you wish it to be installed from.
```

### If not included with the output of your command, run `bundle env` and paste the output below

I can't. It is a security concern to post all of the dependencies, passwords and paths. I can post a reduced version if that would be useful. 

### Note

There is no rack in our local repo.

**Comment by deivid-rodriguez** [comment]
It should be fixed by 2.2.11 which I just released.

**Comment by thebravoman** [comment]
Yes. It is. Perfect :beers:
